### PR TITLE
[CN-1247]-Fix the WAN replication test

### PR DESCRIPTION
**Pull request by cheels**

### CODE REVIEW

File: test/e2e/platform_wan_test.go [hunk 1]
```diff
@@ -455,18 +456,177 @@ var _ = Describe("Hazelcast WAN", Label("platform_wan"), func() {
...
+			}
+			hazelcastSource.Spec.LocalDevices[0].PVC.RequestStorage = &[]resource.Quantity{resource.MustParse(diskSize)}[0]
+			hazelcastSource.Spec.Resources = &corev1.ResourceRequirements{
+				Limits: map[corev1.ResourceName]resource.Quantity{
+					corev1.ResourceMemory: resource.MustParse(totalMemorySize)},
+			}
+			hazelcastSource.Spec.NativeMemory = &hazelcastcomv1alpha1.NativeMemoryConfiguration{
+				Size: []resource.Quantity{resource.MustParse(nativeMemorySize)}[0],
+			}
+
```
Comment (cagric0) [inline test/e2e/platform_wan_test.go:491]: Missing `hazelcastSource.Spec.ClusterName = "source"`?

File: test/e2e/platform_wan_test.go [hunk 2]
```diff
@@ -455,18 +456,177 @@ var _ = Describe("Hazelcast WAN", Label("platform_wan"), func() {
 	})
 
 	Context("Data Sync between separate clusters", func() {
-		It("shouldn't fail in active-passive mode across separate clusters", Serial, Tag(EE|AnyCloud), func() {
-			if !ee {
-				Skip("This test will only run in EE configuration")
+		It("should replicate data for maps with TS and non TS storage in active-passive WAN replication mode", Serial, Tag(EE|AnyCloud), func() {
+			SwitchContext(context1)
+			setupEnv()
+			setLabelAndCRName("hpts-1")
```
Comment (cagric0) [inline test/e2e/platform_wan_test.go:462]:
```suggestion
			setLabelAndCRName("hpwts-1")
```
We are already using `hpts` in the platform tiered storage suite.

File: test/e2e/platform_wan_test.go [hunk 3]
```diff
@@ -455,18 +456,177 @@ var _ = Describe("Hazelcast WAN", Label("platform_wan"), func() {
...
+			})
+			DeletePod(hazelcastTarget.Name+"-0", 0, targetLookupKey)
+			DeletePod(hazelcastTarget.Name+"-1", 0, targetLookupKey)
+			DeletePod(hazelcastTarget.Name+"-2", 0, targetLookupKey)
+			evaluateReadyMembers(targetLookupKey)
+
+			By("trigger WAN sync")
+			SwitchContext(context1)
+			setupEnv()
+			createWanSync(ctx, sourceLookupKey, wanSrc.Name, 2, labels)
```
Comment (cagric0) [inline test/e2e/platform_wan_test.go:604]: I think after the wan sync, we should check the target cluster's map size. WDYT?

**Review by cagric0 — APPROVED** [review]
LGTM! I comment about a few minor issue.

**Review by kutluhanmetin — APPROVED** [review]

**Pull request merged**